torch/ao/quantization/fx/_model_report/model_report_observer.py

```diff
@@ -0,0 +1,77 @@
+import torch
+from torch.ao.quantization.observer import ObserverBase
+
+
+class ModelReportObserver(ObserverBase):
+    r"""This observer is used to record additional information regarding keeping track
+    of S = average_batch_activation_range/epoch_activation_range.
+
+    The purpose of this information is to prepare a report to present to users on whether
+    Dynamic or Static Quantization is more appropriate for their model given the general
+    distributions of their data.
+
+    Note: this tool is meant for FX Graph Mode Quantization
+    """
+
+    def __init__(self):
+        super().__init__(torch.qint8)
```

Comment:
is this dtype not used? maybe we could remove this in the base class as well

Reply:
I don't think it's used, but i need this line or else the submodules cannot be correctly accessed..